hysds_commons/stac_utils.py

```diff
@@ -0,0 +1,126 @@
+import copy
+import os
+import json
+from hysds_commons.log_utils import logger
+
+# load project configuration files
+# load product metadata file
+# read in product_path from _context.json
+# create list for product lineage
```

Comment:
Please review all the comment and clean up any personal notes and comments. You can keep the ones that help explain the code right below it.